__tests__/developers.test.js

```diff
@@ -110,5 +97,18 @@ describe('Developers', () => {
       const res = await getWord(limitWordId, { apiLimit: 2 }, { apiKey: developerRes.body.apiKey });
       expect(res.status).toEqual(403);
     });
+
+    it('should return developer document with correct credentials', async () => {
+      const developerRes = await createDeveloper(developerData);
+      const developerDetails = await getDeveloper({ apiKey: developerRes.body.apiKey });
+      expect(developerDetails.status).toEqual(200);
+      expect(developerDetails.body.message).toEqual('Developer retrieved!');
```

Comment:
instead of checking the `message` field, can you check each field on the returned developer document to see if it's in the right shape with the right data?

Reply:
Done